apps/geo/serializers.py

```diff
@@ -148,3 +153,38 @@ class Meta:
             'region_title', 'admin_level_level', 'admin_level_title',
             'parent'
         )
+
+
+class RegionGqSerializer(ProjectPropertySerializerMixin, UserResourceSerializer):
+    project = IntegerIDField()
```

Comment:
Also, @susilnem can you assist @sudan45 with the Primary key field.
With Primary key field, we don't need to do objects.get.
But let's make sure the graphQl type generated is ID instead of int

Also, the field is optional. Please check the graphql schema

Reply:
If this field is mandatory, then Using PrimaryKeyField would be better.